compose/cli/main.py

```diff
@@ -1327,6 +1328,9 @@ def run_one_off_container(container_options, project, service, options, toplevel
 
     project.initialize()
 
+    # Ensure that run command remains one-off (issue #6302)
+    container_options['restart'] = parse_restart_spec('no')
```

Comment:
We can just set this to `None`, as no restart is the default. This also spares us the additional import.